sources/api/migration/migrations/v1.1.3/kubelet-cpu-manager-state/src/main.rs

```diff
@@ -0,0 +1,59 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::{error, migrate, Migration, MigrationData, Result};
+use snafu::ResultExt;
+use std::fs;
+use std::io;
+use std::path::Path;
+use std::process;
+
+const SETTING: &str = "settings.kubernetes.cpu-manager-policy";
+const CPU_MANAGER_POLICY_CHECKPOINT: &str = "/var/lib/kubelet/cpu_manager_state";
+/// We remove cpu manager policy checkpoint value if checkpoint value is different from upgrade version
```

Comment:
I think we're removing this checkpoint file always. Not only when it's different.